### PR TITLE
Fix jet ordering in MvaMEtUtilities class 76X

### PR DESCRIPTION
**Pull request by mmarionncern**
Same as #9961 for 76X

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

File: RecoMET/METPUSubtraction/src/MvaMEtUtilities.cc [hunk 1]
```diff
@@ -80,7 +80,8 @@ reco::Candidate::LorentzVector MvaMEtUtilities::jetP4(const std::vector<reco::PU
   reco::Candidate::LorentzVector retVal(0.,0.,0.,0.);
   if ( idx < jets.size() ) {
     std::vector<reco::PUSubMETCandInfo> jets_sorted = jets;
-    std::sort(jets_sorted.begin(), jets_sorted.end()); 
+    std::sort(jets_sorted.begin(), jets_sorted.end()); //from lower pt to larger pt
+    std::reverse(jets_sorted.begin(), jets_sorted.end()); //from larger pt to lower pt
```
Comment (slava77) [inline RecoMET/METPUSubtraction/src/MvaMEtUtilities.cc]: wouldn't just
``` std::sort(jets_sorted.rbegin(), jets_sorted.rend()); ``` do what's needed?

Comment (mmarionncern) [inline RecoMET/METPUSubtraction/src/MvaMEtUtilities.cc]: Indeed, corrected

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by cvuosalo** [comment]
+1

For #9963 b71d572e3b20f672f306bc054458b15cb067a46a

Small change to improve sorting of jets for MET MVA. It should not have an effect on standard sequences. The 74X and 75X versions of this PR, #9961 and #9962, have already been approved by Reco.

The code change is satisfactory, and Jenkins [tests](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-9963/5609/summary.html) against baseline CMSSW_7_6_X_2015-07-01-2300 show no significant differences, as expected.

**Comment by davidlange6** [comment]
+1

**Pull request merged**